src/components/DevTools/DevTools.ctypes.tsx

```diff
@@ -35,7 +35,7 @@ class BsCType {
     return cType
       .store(ownerIdentity)
       .then((value: any) => {
-        const cTypeWrapper: ICType = {
+        const cTypeWrapper: any = {
```

Comment:
The only problem I am currently facing, I am able to find a way around this without changing the whole state. As we take the sdk.CType as the state. Which has been changed by removing the metadata. 

It would require me to take the demo client apart and feels unnecessary to complete the ticket. 

I tested the functionality and if someone uses the utilities to create cTypes, I believe it encounters a problem.

Reply:
@LeonFLK changed it to `CTypeWithMetadata`